api/ccapi/include/model.h

```diff
@@ -91,19 +77,19 @@ class Model {
   virtual int setProperty(std::vector<std::string> values) = 0;
 
   /**
-   * @brief     Initialize Network. This should be called after set all
-   * hyperparameters.
+   * @brief     Compile Network. This should be called before initialize
    * @retval #ML_ERROR_NONE Successful.
    * @retval #ML_ERROR_INVALID_PARAMETER invalid parameter.
    */
-  virtual int initialize() = 0;
+  virtual int compile() = 0;
 
   /**
-   * @brief     Compile Network. This should be called before initialize
+   * @brief     Initialize Network. This should be called after setting the
+   * property and compiling. hyperparameters.
```

Comment:
I wonder about the last word `hyperparameters.`

Reply:
done.